Resources/Prototypes/_Shitmed/Entities/Structures/Walls/walls.yml

```
@@ -84,3 +106,16 @@
   - type: Appearance
   #- type: Reflect # Mono - remove reflect
   #  reflectProb: 1
+  - type: Destructible # Mono - make them destructible
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)